niworkflows/interfaces/nibabel.py

```diff
@@ -89,3 +89,68 @@ def _run_interface(self, runtime):
         maskimg.to_filename(self._results['out_mask'])
 
         return runtime
+
+
+class _FourToThreeInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='input 4d image')
+
+
+class _FourToThreeOutputSpec(TraitedSpec):
+    out_files = OutputMultiObject(File(exists=True),
+                                     desc='output list of 3d images')
+
+
+class FourToThree(SimpleInterface):
+    """Split a 4D dataset along the last dimension
+    into a series of 3D volumes."""
+
+    input_spec = _FourToThreeInputSpec
+    output_spec = _FourToThreeOutputSpec
+
+    def _run_interface(self, runtime):
+        filenii = nb.load(self.inputs.in_file)
+        if len(filenii.shape) != 4:
+            raise RuntimeError('Input image (%s) is not 4D.' % filenii)
+
+        files_3d = nb.four_to_three(filenii)
+        out_files = []
+        for i, file_3d in enumerate(files_3d):
+            out_file = fname_presuffix(in_file, suffix="_tmp_{}".format(i))
```

Comment:
We are Python +3.6, let's use f-strings and make it a bit bids-y:
```suggestion
            out_file = fname_presuffix(in_file, suffix=f"_idx-{i:03}")
```

Reply:
sounds good